flash/website/models.py

```diff
@@ -2,18 +2,28 @@
 from django.contrib.auth.models import User
 import random
 
-NUM_BOXES = 3
-BOXES = range(1, NUM_BOXES + 1)
+
+class Box(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    box_number = models.IntegerField()
+
+    @staticmethod
+    def get_unique_boxes(user):
```

Comment:
Does this method return unique boxes or user's boxes? If the latter - get_user_boxes would be a better name